### DISCUSSION TITLE
No puedo crear canales en el administrador para un grupo tipo workflow

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#195] [4. Bug Reports] [answered]
Buenas,

Anteriormente podía crear canales en la vista de canales de un grupo tipo workflow (arriba al lado de la lupa estaba el símbolo más "+"). Ahora en esta implementación (nueva compañía) no veo esa función.
Sugerencia: La [documentación](https://doc.cotalker.com/docs/documentation/admin/workflows/admin_workflow_groups#secondary-actions) podría incluir cómo configurar un secondary action para crear una tarea con un ejemplo (levantar un formulario) ya que esta configuración es necesaria para dar uso a las nuevas funcionalidades y no está totalmente explícito actualmente.

![Screen Shot 2021-05-17 at 14 35 41](https://user-images.githubusercontent.com/12589784/118539872-f490d880-b71d-11eb-8405-a0136257f4fa.png)

Saludos.

**Comment by gmoplaza** [comment, marked as answer]
@SeIturra ya no estamos permitiendo crear canales en grupos de tipo workflow. Para que necesitas un canal? Quizás podemos encontrar alguna solución a lo que necesitas.

**Reply by seiturra** [reply]
@gmoplaza Gracias! Si es una feature no hay problema. Era para hacer el típico canal para crear tareas sólo por la costumbre. Tengo el FAB button configurado para crear la tarea por lo que no necesitaría crear ese canal.

La sugerencia para la documentación es para que en los pasos de Secondary Actions o en los pasos para configurar un workflow se incluya que se debe generar previamente el initial survey del workflow, hacerlo público y ese link generado incluirlo como Action.

**Reply by NicolasPLG** [reply]
hola @gmoplaza entonces cómo se podría hacer si el cliente quiere crear un grupo de conversaciones en el mismo workflow?. Se puede crear un canal de conversación en: "conversaciones / private", pero al hacer eso, el cliente tendría que salir del workflow para ir a conversaciones y consultar por algo antes de crear una tasks, en vez de verlo en el mismo workflow.

Otra forma que se podría necesitar un canal es la consulta de un activo, para ver las última información y ver si creo una tasks de seguimento / reemplazo  / actualización de información. Se qué los datos se pueden ver en el Data Base y actualizarlos desde allí, pero no podrá crear una tasks de forma directa.

Son algunos ejemplos que se me ocurren ahora, pueden existir más. La ventaja que veo la creación de canales es para que el cliente no deba ir a otro lado, para conversar o consultar un registro".

Gracias :>
